packages/swap/src/routes/swap.ts

```diff
@@ -277,6 +280,14 @@ router.get(
       failedAt: failedSwap?.failedAt,
       failedBlockIndex: failedSwap?.failedBlockIndex ?? undefined,
       depositChannelAffiliateBrokers: affiliateBrokers,
+      depositChannelMaxBoostFeeBps: swapDepositChannel?.maxBoostFeeBps,
+      failedBoosts:
+        swapDepositChannel?.failedBoosts.map((failedBoost) => ({
+          failedAtTimestamp: failedBoost.failedAtTimestamp,
+          failedAtBlockIndex: failedBoost.failedAtBlockIndex,
+          amount: failedBoost.amount.toFixed(),
+        })) ?? [],
```

Comment:
this only happens if the liquidity is insufficient, right? would `skippedBoots` be a better term?

Reply:
This one is a bit tricky. We will need it on the frontend to be able to transition to the right view (insufficient liquidity / boost found). Do you think checking against the length of `failedBoosts` is enough ( + checking the `effectiveBoostFeeBps`) to determine the status? 
How do we handle multiple swaps on the frontend? There are no transitions (screen stays on the swap completed card) after the first swap completed right? 
In that case, its safe to just check against the length since we will show the transitions only once.

Reply:
> this only happens if the liquidity is insufficient, right? would `skippedBoots` be a better term?

Yes, if there was not enough liquidity in the pools, there will be a failedBoost entry. At that point, there is still no swap as the deposit is pending finalization (normal swap flow).

Reply:
we dont handle multiple swaps in the swapping frontend at all 🙂

Reply:
so i think for the "not enough liquidity" screen it would be enough to check the length like you said